burnman/data/input_seismic/prem.txt

```diff
@@ -0,0 +1,95 @@
+#Depth(m)	Radius(m)	Pressure(Pa)	Gravity(m/s^2)	Density(g/m^3)	Vp(m/s)	Vs(m/s)	Qk	Qmu
```

Comment:
I've calculated gravity, because although it is in the original paper, it isn't anywhere in the PREM data tables that I have. 
And values have been reinterpolated, and I gave discontinuities a finite width of 1km instead of 0 km, because I think various interpolations didn't deal with having two values for the same depth.